stdlib/asm/collections/mmr.masm

```diff
@@ -104,3 +106,70 @@ export.get
      # stack: [leaf, ...]
   end
 end
+
+#! Load the MMR peak data based on its hash.
+#!
+#! Input: [HASH, mmr_ptr, ...]
+#! Output: [...]
+#!
+#! Where:
+#!  - HASH: is the MMR peak hash, the hash is expected to be padded to an even
+#!    length and to have a minimum size of 16 elements
```

Comment:
nit: I'd probably add (either here or somewhere else in the doc comment) that we expect that the advice map contains certain info under this hash, and if not, this procedure will fail.

Reply:
added